knees4bees/rancid-tomatillos
# Feature/router

## Is this a fix, feature, or refactor?
Refactor/feature

## What is the change?
Adds React Router to mimic a multi-page app.

## What does it fix?
n/a

## Where should the reviewer start?
Start reviewing `App.js` and follow the routes through `Main.js` and `MovieDetail.js`.

## How should this be tested?
Start on the home page and confirm that all elements are present (nav bar, carousel, movie tiles). Click on a movie and confirm that a movie detail page appears with a corresponding url; try this by clicking on one of the carousel images and then again on a regular movie tile. Make sure the browser's forward and back buttons work as they should.

## Relevant ScreenShots
